src/js/libcs/Accessors.js

```diff
@@ -219,6 +221,11 @@ Accessor.setField = function(geo, field, value) {
             geo.pinned = value.value;
         }
     }
+    if (field === "labeled" || field === "labelled") {
```

Comment:
If you have the name mapping in `generalFields`, using a single spelling variant here should suffice, right?